components/icons/fix-svg.js

```diff
@@ -1,6 +1,6 @@
 export function fixSvg(svg) {
 
-	const paths = svg.querySelectorAll('path[fill]');
+	const paths = svg.querySelectorAll('[fill]');
```

Comment:
Haha, I guess the variable doesn't really describe the result very well anymore eh.

Reply:
Good call... will fix.